benchmarks/bench_sample_without_replacement.py

```diff
@@ -205,7 +205,7 @@ def bench_sample(sampling, n_population, n_samples):
     )
     print(25 * "-" + ("|" + "-" * 14) * 1)
     for key, value in arguments.items():
-        print("%s \t | %s " % (str(key).ljust(16), str(value).strip().center(12)))
+        print("%s \t | %s " % (key.ljust(16), value.strip().center(12)))
```

Comment:
I'm not sure if this is equivalent. At least I read the original line as "`key` might not yet be a string". The benchmark scripts aren't automatically tested, so either someone needs to run this to confirm the change is ok or we skip modifying the benchmark (they aren't used that much so spending effort on them is lower bang-for-buck)

Reply:
The code looks like this:
```python
    op = optparse.OptionParser()
    [...]
    (opts, args) = op.parse_args()
    [...]
    arguments = vars(opts)
    [...]
    for key, value in arguments.items():
```
I believe the keys of `opts`, hence `key`, can only be `str`. I had actually extracted and executed the relevant code to check that.